ext/standard/array.c

```diff
@@ -1078,15 +1098,7 @@ PHP_FUNCTION(prev)
 	zend_hash_move_backwards(array);
```

Comment:
Does this need tweaking when the previous element is undef? Otherwise, we'll just immediately advance to the current element again.

Reply:
I don't understand your question. We're moving 1 position backwards, and then call `ia_return_current` with `forward_direction == false`, so when the previous element is undef we just keep going backwards.

Reply:
Ah, makes sense. I clearly didn't look at this close enough.